# weather board sensor msg

#header
Header header

# si1132
float64 UV_index
float64 visible       # (Lux)
float64 IR	      # (Lux)

# bme280
float64 temperature   # (C)
float64 humidity      # relative humidity (%)
float64 pressure      # (hPa)
float64 altitude      # (m)

